src/python/weblyzard_api/client/recognize/ng.py

```diff
@@ -65,7 +74,45 @@ def __init__(self, url=WEBLYZARD_API_URL, usr=WEBLYZARD_API_USER,
         MultiRESTClient.__init__(self, service_urls=url, user=usr, password=pwd,
                                  default_timeout=default_timeout)
         self.profile_cache = []
+    
+    
+    @classmethod
+    def convert_document(cls, xml, version='0.4'):
```

Comment:
convert_document_to_dict would be better I think (at first reading I thought between different XML versions)